scripts/collect_segmentation_intermediates.py

```diff
@@ -0,0 +1,99 @@
+"""
+Input:
+- <chainsaw/protein.tsv>: Segmentation by chainsaw
+- <merizo/protein.tsv>:   Segmentation by merizo
+- ...
+
+Output:
+- <segmentation.tsv>: A single tsv with all the different segmentations
+"""
+
+import sys
+import os
+import csv
+import json
+import re
+from pathlib import Path
+
+if len(sys.argv) < 2:
+    print(f"USAGE: python {os.path.basename(__file__)} [inputs] <output>")
+    sys.exit(1)
+
+def main():
+    input_files = sys.argv[1:-1]
+    output_file = sys.argv[-1]
+
+    columns = ['index', 'method', 'domain_count', 'chopping']
+    segmentations = []
+    index = 1
+
+    for path in input_files:
+        if '/chainsaw/' in path:
+            rows = read_csv_rows(path, delimiter='\t')
+            data = rows[0]
+            segmentations.append((index, "chainsaw", data['ndom'], data['chopping']))
+            index += 1
+        elif '/merizo/' in path:
+            pass
+            # Ignore for now, fails on these PDBs
```

Comment:
It might be failing, because I'm running it on the pdb with no secondary structure. For now, it doesn't matter too much.